include/pgduckdb/pg/relations.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "pgduckdb/pg/declarations.hpp"
+
+namespace pgduckdb {
+
+TupleDesc PDRelationGetDescr(Relation relation);
```

Comment:
Why does only this one need tho PD prefix? does `::RelationGetDescr` not work for macros?

Reply:
No, macros are basically inline (as text) directly in the source code.

So if you have something like this:
```c++
#define foo(a, b) something_foo(a, b)
```

And you try to declare a function of the same name:
```c++
// In the header
void foo(int a, int b);

// In the source file
#include "my_foo_macro.h"

void foo(int a, int b) {
  ::foo(a, b);
}
```

Then the pre-processor will transform the source file to:
```c++
void foo(int a, int b) {
  something_foo(a, b)(a, b);
}
```

Which sadly doesn't compile.

We should be able to do this:
```C++
void foo(int a, int b) {
  foo
}
```

As long as the parameter match, but that is super hacky and not very readable IMHO